src/api/data/cache/cache_test.go

```diff
@@ -101,13 +107,14 @@ func TestCachedChartsRefreshErrorPropagation(t *testing.T) {
 }
 
 func TestCachedChartsRefreshErrorDownloadingPackage(t *testing.T) {
-	chartDataExistsOrig := chartDataExists
-	defer func() { chartDataExists = chartDataExistsOrig }()
-	chartDataExists = func(chart *models.ChartPackage) (bool, error) { return false, nil }
-	downloadAndExtractChartTarballOrig := downloadAndExtractChartTarball
-	defer func() { downloadAndExtractChartTarball = downloadAndExtractChartTarballOrig }()
-	knownError := errors.New("error on downloadAndExtractChartTarball")
-	downloadAndExtractChartTarball = func(chart *models.ChartPackage) error {
+	ChartDataExistsOrig := charthelper.ChartDataExists
```

Comment:
I'm new on `Go`, so sorry about the question :P. Why do you capitalize the first letter of variables? Is this a convention?